fontforge/parsettf.c

```diff
@@ -2735,6 +2735,14 @@ return( NULL );
 		++i;
 	    }
 	    --i;
+	} else if ( (offsets[i+1]-offsets[i])==UINT32_MAX ) {
```

Comment:
This is a weirdly specific problem. Normally one would worry about a more general class of strings that are too large. Here you're finding the offset difference is exactly UINT32_MAX. That strongly implies (to me, anyway) that some other code is causing the difference to be that value, and adding this fix would just disguise that problem.

Before we do this we should at least try to figure out why we're seeing such a specific symptom. Do you have an example file?